lib/routes/v1/repos.js

```diff
@@ -72,15 +72,11 @@ module.exports = app => {
 			type: request.query.type,
 			status: request.query.status,
 			supportEmail: request.query.supportEmail,
-			search: request.query.q
+			search: request.query.q,
+			origamiVersion: request.query.origamiVersion,
 		};
-		const hasFilters = (
-			filters.brand ||
-			filters.type ||
-			filters.status ||
-			filters.supportEmail ||
-			filters.search
-		);
+
+		const hasFilters = Object.values(filters).filter(Boolean).length >= 1;
```

Comment:
`.filter(...).length >= 1` could be rewritten to use the `some` method instead.

```suggestion
		const hasFilters = Object.values(filters).some(Boolean);
```